timeseries/src/autogluon/timeseries/trainer/abstract_trainer.py

```diff
@@ -160,28 +117,10 @@ def load_model(
 
         return type_.load(path=os.path.join(self.path, path_), reset_paths=self.reset_paths)
 
-    def construct_model_templates(self, hyperparameters: Union[str, Dict[str, Any]], **kwargs):
-        raise NotImplementedError
-
-    # FIXME: Copy pasted from Tabular
-    def get_minimum_model_set(
-        self, model: Union[str, AbstractTimeSeriesModel], include_self: bool = True
-    ) -> List[str]:
-        """Gets the minimum set of models that the provided model depends on, including itself.
-        Returns a list of model names
-        """
-        if not isinstance(model, str):
-            model = model.name
-        minimum_model_set = list(nx.bfs_tree(self.model_graph, model, reverse=True))
-        if not include_self:
-            minimum_model_set = [m for m in minimum_model_set if m != model]
-        return minimum_model_set
-
-    def get_models_info(self, models: Optional[List[str]] = None) -> Dict[str, Any]:
-        if models is None:
-            models = self.get_model_names()
+    def get_models_info(self, models: Optional[List[str | AbstractModel]] = None) -> Dict[str, Any]:
```

Comment:
What is the reason for the difference between this method and `AbstractTrainer.get_models_info`?

Reply:
will be reconciled in the follow-up PR